modules/openxr/editor/openxr_action_map_editor.cpp

```diff
@@ -288,17 +290,17 @@ void OpenXRActionMapEditor::_load_action_map(const String p_path, bool p_create_
 	} else if (p_create_new_if_missing) {
 		action_map.instantiate();
 		action_map->create_default_action_sets();
-		action_map->set_path(p_path);
+		action_map->set_path(path);
 
 		// Save it immediately
-		err = ResourceSaver::save(action_map, p_path);
+		err = ResourceSaver::save(action_map, path);
 		if (err != OK) {
 			// Show warning but continue.
-			EditorNode::get_singleton()->show_warning(vformat(TTR("Error saving file %s: %s"), edited_path, error_names[err]));
+			EditorNode::get_singleton()->show_warning(vformat(TTR("Error saving file %s: %s"), path, error_names[err]));
```

Comment:
This change isn't strictly related to fixing this bug, but the original code here was incorrect: it was using `edited_path` before we've set `edited_path`, so it would show either `""` or the previous path. We're saving to `path`, so that is what should be shown in the error dialog.